uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/models/instances/leave-request-instance.js

```diff
@@ -247,9 +269,53 @@ define([
          * @param {string} key - The property name
          */
         toAPIFilter: function (result, __, key) {
-          if (!_.includes(['balance_change', 'dates', 'comments'], key)) {
+          if (!_.includes(['balance_change', 'dates', 'comments', 'uploader', 'files'], key)) {
             result[key] = this[key];
           }
+        },
+
+        /**
+         * Loads file attachments associated with this leave request
+         *
+         * @return {Promise}
+         */
+        loadAttachments: function () {
+          return LeaveRequestAPI.getAttachments(this.id)
+            .then(function (attachments) {
+              this.files = attachments;
+            }.bind(this));
+        },
+
+        /**
+         * Sets the flag to mark file for deletion. The file is not yet deleted
+         * from the server.
+         */
+        deleteAttachment: function (attachmentID) {
```

Comment:
This function is not necessary here, it can be handled in the `controller`calling this model as done for comment. 
https://github.com/civicrm/civihr/blob/211ae872153157e9ca6ceabea045a00c8afe2d51/uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js#L282-L282